addons/api/mirage/scenarios/ipc.js

```diff
@@ -151,6 +137,14 @@ export default function initializeMockIPC(server, config) {
    */
   if (config['ember-cli-mirage'].enabled && !isTesting) {
     const mockIPC = new MockIPC();
+
+    if (!config.isElectron) {
+      // Add these handlers for non electron environments, otherwise just
+      // pass through and use the actual electron handlers
+      mockIPC.hasMacOSChrome = () => false;
+      mockIPC.showWindowActions = () => true;
```

Comment:
I had added it back in but thinking back on this, shouldn't we have our dev testing match what the user will have and in this case they won't see those actions on mac?

Reply:
We talked a bit about this in zoom, ideally we would want some sort of feature flag to be able to turn these on or off, but for now we're actually going to have `hasMacOSChrome` always true and `showWindowActions` always true in mocks.

It's probably better to be consistent for now rather than having some handlers mocked and others not